tests/FSharp.Compiler.ComponentTests/Conformance/GeneratedEqualityHashingComparison/Basic/Basic.fs

```diff
@@ -62,15 +62,12 @@ module Basic =
             (Error 344, Line 13, Col 6, Line 13, Col 7, "The struct, record or union type 'R' has an explicit implementation of 'Object.GetHashCode' or 'Object.Equals'. You must apply the 'CustomEquality' attribute to the type")
             (Warning 345, Line 13, Col 6, Line 13, Col 7, "The struct, record or union type 'R' has an explicit implementation of 'Object.GetHashCode'. Consider implementing a matching override for 'Object.Equals(obj)'")
             (Error 359, Line 13, Col 6, Line 13, Col 7, "More than one override implements 'GetHashCode: unit -> int'")
-            (Error 54, Line 13, Col 6, Line 13, Col 7, "This type is 'abstract' since some abstract members have not been given an implementation. If this is intentional then add the '[<AbstractClass>]' attribute to your type.")
```

Comment:
Before we were raising an error in records and unions, which is incorrect. So we are cleaning up here.
```fsharp
type R =
  { a : int; b : string }
  override x.GetHashCode() = x.a + x.b.Length

type U = 
  | A | B
  override x.GetHashCode() = 0

type S = 
  struct
    val mutable a : int
    override x.GetHashCode() = x.a
  end
```